### PR TITLE
Add Numix Gtk theme

### PR DESCRIPTION
**Pull request by bil-elmoussaoui**

### CODE REVIEW

File: org.gtk.Gtk3theme.Numix.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "id": "org.gtk.Gtk3theme.Numix",
+    "branch": "3.22",
+    "runtime": "org.gnome.Sdk",
+    "build-extension": true,
+    "sdk": "org.gnome.Sdk",
+    "runtime-version": "3.26",
+    "appstream-compose": false,
+  "separate-locales": false,
```
Comment (TingPing) [inline org.gtk.Gtk3theme.Numix.json]: Whitespace nitpick.

File: org.gtk.Gtk3theme.Numix.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "modules": [{
+            "name": "Numix",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -dm755 /usr/share/runtime/share/themes/Numix/gtk-3.0",
+                "cp -aL gtk-3.20/* /usr/share/runtime/share/themes/Numix/gtk-3.0",
+                "cp -a index.theme /usr/share/runtime/share/themes/Numix"
+            ],
+            "sources": [{
+                    "type": "archive",
```
Comment (TingPing) [inline org.gtk.Gtk3theme.Numix.json]: Inconsistent whitespace here too.

**Comment by TingPing** [comment]
bot, build org.gtk.Gtk3theme.Numix

**Comment by bil-elmoussaoui** [comment]
Fixed

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/org.gtk.Gtk3theme.Numix

**Pull request closed without merging**